website/source/docs/guides/backup.html.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: "docs"
+page_title: "Cluster Backups"
+sidebar_current: "docs-guides-backups"
+description: |-
+  Consul provide the snapshot tool for backing up and restoring data. In this guide you will learn how to use both.
+---
+
+# Cluster Backups
+
+Creating server backups is an important step in production deployments. Backups provide a mechanism for the server to recover from an outage (network loss, operator error, or a corrupted data directory). All agents write to the `-data-dir` before commit. This directory persists the local agent’s state and - in the case of servers -  it also holds the Raft information.
```

Comment:
Backups are also incredibly important for upgrades since it's often impossible to downgrade in-place due to changes in the state store format and the only way back is to restore from a backup.

We don't need tons of detail on that here but just wanted to mention incase it's worth including briefly in that list?

Reply:
```suggestion
Creating server backups is an important step in production deployments. Backups provide a mechanism for the server to recover from an outage (network loss, operator error, or a corrupted data directory). All servers write to the `-data-dir` before commit on write requests. The same directory is used on client agents to persist local state too, but this is not critical and can be rebuilt when recreating an agent. Local client state is not backed up in this guide and doesn't need to be in general, only the server's Raft store state.
```